### PR TITLE
Top product api2

### PR DESCRIPTION
**Pull request by moriyahazan**

### CODE REVIEW

**Comment by moriyahazan** [comment]
Hi @sara1386,
Please see my updated code with spring Mongo db aggregation, (The query you sent me didn't work, I don't know why)
You can generate data by calling to fill() method I committed (copied from another team)
please generate data only via the application and not directly by the database

if everything seems ok for you please merge to main. 

thanks 
and have a nice day

File: src/main/java/com/sap/ordermanagergreen/service/GraphService.java [hunk 1]
```diff
@@ -40,60 +32,47 @@ public class GraphService {
...
+        private int year;
+        private int month;
+        private Product product;
+        private int totalQuantity;
+
+    }
+
+    public List<MonthlyProductSalesResult> getMonthlyProductSales() {
+
+
```
Comment (geniuscodes) [inline src/main/java/com/sap/ordermanagergreen/service/GraphService.java:62]: extra line

**Review by geniuscodes — CHANGES_REQUESTED** [review]

**Review by sara1386 — APPROVED** [review]
it works. thanks!
and there was really an issue with the data, fill() solved it.

**Comment by moriyahazan** [comment]
great
please merge to main

‫בתאריך יום א׳, 20 באוג׳ 2023 ב-15:29 מאת ‪Sara Leah‬‏ <‪
***@***.***‬‏>:‬

**Pull request merged**